src/main/java/com/example/fooddelivery/service/UserService.java

```diff
@@ -0,0 +1,27 @@
+package com.example.fooddelivery.service;
+
+import com.example.fooddelivery.model.User;
+import com.example.fooddelivery.repository.UserRepository;
+
+/**
+ * UserService for user operations.
+ */
+public interface UserService {
+
+    /**
+     * Get user from DB
+     *
+     * @param email user email
+     * @param password user password
+     * @return {@link User} entity
+     */
+    User getUserByEmailAndPassword(String email, String password);
```

Comment:
Same thing like `Save` method from below. You only use service between facade and repository without any other custom logic. Maybe is better to call repository directly from the facade, *maybe*.